src/QMCDrivers/DMC/DMCBatched.cpp

```diff
@@ -232,19 +233,20 @@ void DMCBatched::advanceWalkers(const StateForThread& sft,
       std::transform(drifts.begin(), drifts.end(), log_gb.begin(),
                      [oneover2tau](auto& drift) { return -oneover2tau * dot(drift, drift); });
 
-      std::transform(ratios.begin(), ratios.end(), prob.begin(),
-                     [](auto ratio) { return std::norm(ratio); });
+      for (int iw = 0; iw < num_walkers; ++iw)
+      {
```

Comment:
Could you remove the `{}` and save two lines when you make a commit for other changes..